### DISCUSSION TITLE
Is there something like a "black hole" device that can be used for marking an interface as connected?

### DISCUSSION THREAD

**Discussion by dougthor42** [netbox-community/netbox#6402] [Q&A] [answered]
Is there something like a "black hole" device that can be used for marking an interface as connected?

Or perhaps someone else has an alternative workaround?

I can't think of any search term other than "black hole" for this concept, so maybe someone else already had this idea...

### What I'm trying to solve:

I have a firewall that I have to connect a device to. I don't care about this device at all - all I care is that the firewall's interface is marked as "connected".

Previously I'd just add a Description to the interface and call it a day, but that's starting to become a problem as we grow.

![image](https://user-images.githubusercontent.com/5386897/118013343-dfbfd980-b306-11eb-9d01-bc158075ef37.png)

### Ideal Solution

A "dummy" device that can **accept any number of connections** on **any type of interface**. This would allow users to define the cable and mark the interface as connected, without having to create a Device Type and Device for the connection.

![image](https://user-images.githubusercontent.com/5386897/118013213-bc952a00-b306-11eb-8b57-2be12531a876.png)

### Current Workaround

I guess I could make a device that has a large number of interfaces, front ports, back ports, etc., but that sounds unwieldy and would also mess with tools that map the L1 connections based on the cable mapping in Netbox.

**Comment by jeremystretch** [comment, marked as answer]
NetBox v2.11 introduced the ability to mark interfaces as connected without actually attaching a cable (see #3648). That sounds like what you're after.

**Reply by dougthor42** [reply]
That's exactly what I'm looking for, thanks!

I'm still on v2.10.1. I guess I should have read the changelogs first... sorry about that!
